packages/core-data/src/entities.js

```diff
@@ -313,6 +313,7 @@ async function loadPostTypeEntities() {
 			name
 		);
 		const namespace = postType?.rest_namespace ?? 'wp/v2';
+		const syncedProperties = new Set( [ 'blocks' ] );
```

Comment:
I don't understand why we need this change here? It feels very ad-hoc?

Reply:
I don't feel like this addresses my question though. I guess what I'm asking is that before making "blocks" a synced property for a given post type entity, we should first ensure the post type in question supports "content" (which we can do in the supports array no?)

Reply:
Anyway, it's not a huge concern though but we should be thinking about post types that don't have "content" or "blocks".